vsphere/metadata.csv

```diff
@@ -1,156 +1,287 @@
+vsphere.datastore.numberReadAveraged.avg,gauge,,command,,Average number of read commands issued per second to the datastore,0,vsphere,numberReadAveraged avg
+vsphere.datastore.numberWriteAveraged.avg,gauge,,,,Average number of write commands issued per second to the datastore during the collection interval,0,vsphere,datastore nb write avg
+vsphere.datastore.read.avg,gauge,,kibibyte,second,Rate of reading data from the datastore,0,vsphere,datastore read avg
+vsphere.datastore.siocActiveTimePercentage.avg,gauge,,percent,,Percentage of time Storage I/O Control actively controlled datastore latency,-1,vsphere,siocActiveTimePercentage avg
+vsphere.datastore.sizeNormalizedDatastoreLatency.avg,gauge,,microsecond,,Storage I/O Control size-normalized I/O latency,-1,vsphere,sizeNormalizedDatastoreLatency avg
+vsphere.datastore.throughput.contention.avg,gauge,,millisecond,,Average amount of time for an I/O operation to the datastore or LUN across all ESX hosts accessing it.,0,vsphere,datastore thgpt content avg
+vsphere.datastore.throughput.usage.avg,gauge,,kibibyte,second,The current bandwidth usage for the datastore or LUN.,0,vsphere,datastore thgpt usage avg
+vsphere.datastore.totalReadLatency.avg,gauge,,millisecond,,Average amount of time for a read operation from the datastore,-1,vsphere,totalReadLatency avg
+vsphere.datastore.totalWriteLatency.avg,gauge,,millisecond,,Average amount of time for a write operation from the datastore,-1,vsphere,totalWriteLatency avg
+vsphere.datastore.write.avg,gauge,,kibibyte,second,Rate of writing data to the datastore,0,vsphere,datastore write avg
+vsphere.disk.busResets.sum,gauge,,command,,Number of SCSI-bus reset commands issued,0,vsphere,disk busResets sum
+vsphere.disk.capacity.contention.avg,gauge,,percent,,"The amount of storage capacity overcommitment for the entity, measured in percent.",0,vsphere,disk cap content avg
+vsphere.disk.capacity.latest,gauge,,kibibyte,,Configured size of the datastore,0,vsphere,disk capacity latest
+vsphere.disk.capacity.provisioned.avg,gauge,,kibibyte,,"Provisioned size of the entity",0,vsphere,disk cap prov avg
+vsphere.disk.capacity.usage.avg,gauge,,kibibyte,,"The amount of storage capacity currently being consumed by or on the entity.",0,vsphere,disk cap usage avg
+vsphere.disk.commands.sum,gauge,,command,,Number of SCSI commands issued,0,vsphere,disk commands sum
+vsphere.disk.commandsAborted,gauge,,occurrence,,Number of SCSI commands aborted,-1,vsphere,disk cmd abort
+vsphere.disk.commandsAborted.sum,gauge,,command,,Number of SCSI commands aborted,-1,vsphere,disk commandsAborted sum
+vsphere.disk.commandsAveraged.avg,gauge,,command,second,Average number of SCSI commands issued per second,0,vsphere,disk commandsAveraged avg
+vsphere.disk.deltaused.latest,gauge,,kibibyte,,Storage overhead of a virtual machine or a datastore due to delta disk backings,-1,vsphere,disk deltaused latest
```

Comment:
It seems that the unit is not `kibibyte` but `kilobyte`, that might be possibly misleading:

```
   (vim.PerformanceManager.CounterInfo) {
      dynamicType = <unset>,
      dynamicProperty = (vmodl.DynamicProperty) [],
      key = 266,
      nameInfo = (vim.ElementDescription) {
         dynamicType = <unset>,
         dynamicProperty = (vmodl.DynamicProperty) [],
         label = 'Space actually used',
         summary = 'Amount of space actually used by the virtual machine or the datastore',
         key = 'used'
      },
      groupInfo = (vim.ElementDescription) {
         dynamicType = <unset>,
         dynamicProperty = (vmodl.DynamicProperty) [],
         label = 'Disk',
         summary = 'Disk',
         key = 'disk'
      },
      unitInfo = (vim.ElementDescription) {
         dynamicType = <unset>,
         dynamicProperty = (vmodl.DynamicProperty) [],
         label = 'KB',
         summary = 'Kilobyte',
         key = 'kiloBytes'
      },
      rollupType = 'latest',
      statsType = 'absolute',
      level = 1,
      perDeviceLevel = 1,
      associatedCounterId = (int) []
   },
```

Reply:
It's actually kibibytes despite the unit name because vSphere 🤷‍♂ 
https://code.vmware.com/apis/704/vsphere/memory_counters.html